src/app/feed/feed-post-icon-row/feed-post-icon-row.component.html

```diff
@@ -1,152 +1,350 @@
-<div class="js-feed-post-icon-row__container fs-14px text-grey5 d-flex justify-content-between unselectable">
+<section>
   <div
-    class="cursor-pointer d-flex align-items-center"
-    (click)="openModal($event)"
-    data-toggle="modal"
-    data-target=".js-feed-post-icon-row__comment-modal"
+    *ngIf="postReactionCounts.Total > 0 && !hideSummary"
+    class="reactions-summary__container d-inline-flex text-secondary pt-10px"
+    (click)="openReactionsDetails($event)"
   >
-    <div>
-      <i-feather name="message-square" class="feed-post-icon-row__icon reply-icon"></i-feather>
-    </div>
-    <div>
-      <span *ngIf="!hideNumbers">{{ postContent.CommentCount }}</span>
+    <ng-container *ngFor="let reaction of postReactionCounts.Counts | keyvalue">
+      <div
+        *ngIf="reaction.value > 0 && allowedReactions.includes(reaction.key)"
```

Comment:
This check `allowedReactions.includes(reaction.key)` will not show a reaction if it is not in allowed reactions list. E.g. if any other app sends something like `{ type: "REACTION", value: "FIRE" }`